packages/bpk-component-accordion/src/BpkAccordionItem.scss

```diff
@@ -32,18 +32,18 @@
     cursor: pointer;
     appearance: none;
 
-    @include bpk-themeable-property(color, --bpk-accordion-color, $bpk-link-color);
+    @include bpk-themeable-property(color, --bpk-accordion-color, $bpk-color-sky-blue-shade-03);
```

Comment:
This has changed due to link beinging used in multiple places and in the new brand is not the same colour

Reply:
but surely `$link-colour === skyBlue ===` what we want here?

Reply:
So maybe but there is heavier use of this in Navbar and the color can't be skyBlue there as it will clash with the background